examples/openshift-ovs-vsctl.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: extensions/v1beta1
```

Comment:
why do we put this under "examples", this is part of the "installation" of the product. where do we place it for kubevirt?

Reply:
In KubeVirt, generated examples are not kept under the repo IIUIC, you need to call make. Not 100% sure.

We keep it under examples/ since user might want to change image source/version, namespaces etc.

Reply:
in kubevirt, it seems like the "installation" manifest is generated based on a template:
https://github.com/kubevirt/kubevirt/blob/8112d64eda725594210ca7f0255a1343b6d4904e/manifests/release/kubevirt.yaml.in
and generated under the _out directory

Reply:
@yuvalif would you prefer to keep them in the repo like this PR proposes or in temporary _out then?

Reply:
I'm not even sure how the template is parametrized in kubevirt...
So, I think "examples" are good. maybe add a readme which explain what in the yaml could be changed? or maybe comments in the yaml?